File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int dup = x;
        int rev = 0;
        while(x>0){
            int last = x%10;
            rev = 10*rev+last;
            x=x/10;
        }
        if(rev==dup) return true;
        return false;
    }
}

File: 0009-palindrome-number/0009-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    let orig = x;
    let rev=0;
    while(x>0){
        rev=(rev*10)+(x%10);
        x=Math.floor(x/10);
    }
    return orig==rev;
};